spring-cloud-dataflow-server-core/src/test/java/org/springframework/cloud/dataflow/server/service/impl/TaskServiceUtilsTests.java

```diff
@@ -309,6 +309,48 @@ public void testImagePullSecretNullCTRProperties() {
 				taskDeploymentProperties.containsKey("deployer.composed-task-runner.kubernetes.imagePullSecret"));
 	}
 
+	@Test
+	public void testUseUserAccessTokenFromCTRProps() {
```

Comment:
We should also have a test that  test if the user sets access token to false.   i.e. composedTaskRunnerConfigurationProperties.setUseUserAccessToken(false);

Reply:
added